docs/main/Help/migration-guide.md

```diff
@@ -0,0 +1,76 @@
+# Migrating from SecretAgent
+
+If you're migrating a script from SecretAgent, you can expect to find:
+
+- a more developer-friendly set of APIs (like our [DomExtenders](/docs/basic-interfaces/dom-extenders))
```

Comment:
No AwaitedDomExtenders?

Reply:
I could go either way on this. In any case, I think using the migration guide to drive platform-wide name changes seemed like too much for this guide PR.

Reply:
I'm not suggesting a change to the code, just wondering how we should communicate about it in writing.

Reply:
Ok, went ahead with this.